src/Workspaces/MSBuild/BuildHost/Rpc/Contracts/MSBuildLocation.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.Serialization;
+
+namespace Microsoft.CodeAnalysis.MSBuild;
+
+[DataContract]
+internal sealed class MSBuildLocation(string path, string version)
+{
+    [DataMember(Order = 0)]
+    public string Path { get; } = path;
```

Comment:
Should this have a comment to what type of path this is? i.e. to msbuild.exe, the path that came from MSBuildLocator, etc.